sdk/core/core-rest-pipeline/src/policies/bearerTokenAuthenticationPolicy.ts

```diff
@@ -43,7 +43,7 @@ export function bearerTokenAuthenticationPolicy(
   return {
     name: bearerTokenAuthenticationPolicyName,
     async sendRequest(request: PipelineRequest, next: SendRequest): Promise<PipelineResponse> {
-      const { token } = await cycler.getToken(scopes, {
```

Comment:
One option for alleviating the naming concern here is that we could simply replace `const cycler = createTokenCycler(credential);` with `const getToken = createTokenCycler(credential);`

Reply:
I like that!